### PR TITLE
Chocolatey distribution for Windows

### PR DESCRIPTION
**Pull request by Starz0r**
I'm currently the package maintainer for the Chocolatey distribution on Windows. Saw one of the development goals was that, so I thought I add it to the README and knock it out of the goals for you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Nyrna
 
+[![Packaging status](https://repology.org/badge/tiny-repos/nyrna.svg)](https://repology.org/project/nyrna/versions)
+[![latest packaged version(s)](https://repology.org/badge/latest-versions/nyrna.svg)](https://repology.org/project/nyrna/versions)
```
Comment (Merrit) [inline README.md:4]: I think it looks better without these two, the verticle one under `# Installing` looks better too.
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -10,6 +13,8 @@ Nyrna works on Linux with X11 and Microsoft Windows (tested on Windows 10).
 
 # Installing
 
+[![Packaging status](https://repology.org/badge/vertical-allrepos/nyrna.svg)](https://repology.org/project/nyrna/versions)
```
Comment (Merrit) [inline README.md:16]: :+1:

File: README.md [hunk 3]
```diff
@@ -22,6 +27,12 @@ A package is available as [nyrna](https://github.com/BlueManCZ/edgets/tree/maste
 
 - `layman --add edgets && emerge --ask nyrna`
 
+## Microsoft Windows
+
+Available in the [Chocolatey Community Packages repository](https://chocolatey.org/packages/nyrna)
```
Comment (Merrit) [inline README.md:32]: Just needs a period to fit with the rest of the styling

```suggestion
Available in the [Chocolatey Community Packages repository.](https://chocolatey.org/packages/nyrna)
```

File: README.md [hunk 4]
```diff
@@ -69,7 +80,7 @@ I haven't seen an issue like this, however if at any time the hotkey isn't worki
 - Option to minimize before suspend
 - Automatic check for updates
 - AppImage for Linux
-- Chocolatey distribution for Windows
+-   ~~Chocolatey distribution for Windows~~
```
Comment (Merrit) [inline README.md:83]:
```suggestion
- ~~Chocolatey distribution for Windows~~ ✅
```

**Review by Merrit — CHANGES_REQUESTED** [review]
This is great! Never even occurred to me someone might have done this.

Please check over my suggestions :smiley:

**Review by Starz0r — COMMENTED** [review]
Done!

**Review by Merrit — APPROVED** [review]
Very nice, thank you! :heart:

**Pull request merged**